cosmic-app-list/src/app.rs

```diff
@@ -1500,12 +1500,12 @@ impl cosmic::Application for CosmicAppList {
                     }
                     if is_favorite {
                         content = content.push(
-                            menu_button(iced::widget::text(fl!("unfavorite")))
+                            menu_button(iced::widget::text(fl!("unpin")))
                                 .on_press(Message::UnFavorite(desktop_info.id.clone())),
                         )
                     } else if let Some(_) = desktop_info.exec.clone() {
                         content = content.push(
-                            menu_button(iced::widget::text(fl!("favorite")))
+                            menu_button(iced::widget::text(fl!("pin")))
                                 .on_press(Message::Favorite(desktop_info.id.clone())),
```

Comment:
I would still change it to something like `Message::UnpinApp` and `Message::PinApp`

Reply:
Maybe @wash2 can say what would be preferred by the team.
But yes, adding 'App' to the end does make it pretty unambiguous.

Reply:
I think he meant for the Message variant. It should be in sync with the fl macro

Reply:
Yeah, I understood. I'm just waiting for the team say what would be their preferred Message, before making changes.

Reply:
Following discussions, I'll rename most things referencing "favorite", so converting this to draft until done.